### ISSUE TITLE
🐛 Renaming running user tasks causes trouble

### ISSUE THREAD

**Issue by LeoTT** [process-engine/process_engine_runtime#76]
#### Describe your issue

FlowNodeInstances with not existing FlowNodeIds in the database will cause requests for running userTasks to fail with a bad error message.

You might argue that there should not be invalid FlowNodeIds in the first place.
However, the user can easily produce that scenario by starting a FlowNodeInstance and renaming the usertask while it is running.

#### How can others reproduce the issue?

- start a diagram with user task
- leave the user task running. Open the diagram and rename the user task
- try to start the diagram or look at running instances. Either should fail

Requests to `http://localhost:8000/api/management/v1/process_models/diagram_name/user_tasks` then return:
```
Unhandled rejection InternalServerError: Cannot read property 'formFields' of undefined
```

#### Possible solution

Not sure whether this needs a fix in backend or bpmn-studio should just prohibit renaming user tasks when there are running instances with it.

At least the error message can be made more clear.

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by ElRaptorus** [comment]
@LeoTT @5ebastianMeier @Vyperus @cmg-dev @roblen45 @moellenbeck 
I think the question here is, if we really want to allow a user to rename a running UserTask. 
Off hand, I'd say no, because this can cause all kinds of problems, as evidenced by this issue.

It is also very hard to keep track of such a change *during* a process models execution, because a correlation is always hard wired to a specific version of a ProcessModel.
This **may** be addressable when the `Inspect Correlation` feature is finished, but I'd not recommend it.

In my opinion, BPMN studio should not allow to modify running ProcessModels at all.

**Comment by roblen45** [comment]
We should always allow the user to modifiziert the process Model, but in future that should in a new version of model

**Comment by ElRaptorus** [comment]
@roblen45 @LeoTT 
I almost forgot about this issue...

This is actually going to be fixed with the upcoming release of v5.5.0 🙂

**Comment by ElRaptorus** [comment]
This is fixed now.
